src/site/content/en/handbook/markup-code/index.md

```diff
@@ -78,6 +78,16 @@ Show that lines have changed by using the pattern `language/lineNumAdded/lineNum
 </label>
 ```
 
+## Escaping code
+If you need to use double `{` braces in your code examples you will need to
```

Comment:
```suggestion
To keep the parser from interpreting special characters in your code samples (for example, `{{`), escape them using the `{% raw %}{% verbatim %}` shortcode combination:
```
Trying to generalize and make explicit that both shortcodes are needed. Not sure "parser" is the right word. Feel free to just fix and publish if it isn't.

Reply:
I avoided using double curly braces in the example itself because we have to escape them :D

Reply:
Well, <facepalm> for not escaping it (sorry), but I do think it's clearer to just show the double curly braces than write out the phrase. But not a big deal either way.

Reply:
Totally agree. Unfortunately because we publish the handbook to devsite I don't think I'm able to show the double curlies without also showing the unescaped `{% verbatim %}` around them. I thought that might confuse folks so I went with "double `{`"